x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_chart_single_metric.js

```diff
@@ -407,7 +410,9 @@ export class ExplorerChartSingleMetric extends React.Component {
       // Update all dots to new positions.
       dots
         .attr('cx', (d) => lineChartXScale(d.date))
-        .attr('cy', (d) => lineChartYScale(d.value))
+        // Fallback with domain's min value if value is null
+        // To ensure event markers are rendered properly at the bottom of the chart
+        .attr('cy', (d) => lineChartYScale(d.value ? d.value : lineChartYScale.domain()[0]))
```

Comment:
should this be an explicit check for `null`?
`d.value` being `0` will mean it's not used.

Reply:
Good catch 💯
Updated to explicitly check for `null` in [#54e66d2](https://github.com/elastic/kibana/pull/194119/commits/54e66d288ca82e8dc934cae8ba8279225d00cfa3)